components/BarCard/index.js

```diff
@@ -4,6 +4,8 @@ import { Logo } from "../MatchCard";
 export default function BarCard({barWithMatch}){
     return(
         <ListItem>
+        <Flex>
```

Comment:
Maybe you want to give this a more descriptive name, like "StyledContainer"?

Reply:
Thanks David! 
Yeah, good point, I will think of a better name in my next US.